app/app.go

```diff
@@ -154,7 +155,7 @@ type App struct {
 	DistrKeeper         distrkeeper.Keeper
 	GovKeeper           govkeeper.Keeper
 	CrisisKeeper        crisiskeeper.Keeper
-	UpgradeKeeper       upgradekeeper.Keeper // This is included purely for the IBC Keeper. It is not used for upgrading
+	UpgradeKeeper       upgradekeeper.Keeper // Upgrades are set in endblock when signaled
```

Comment:
[nit] Upgrades are set in endblocker only for v3 -> v4. All previous upgrades didn't use the upgrade keeper. Proposal to remove the comment 

```suggestion
	UpgradeKeeper       upgradekeeper.Keeper
```